Change log for 2020-07-06 (#4)
spiir/viia-changelog: README.md

# Viia Changelog
## 2020-07-06

### Added

> - You can now skip the email page, when you are sending a end-user through the onboarding

Reviewer: ```suggestion
- You can now skip the email page, when you are sending a end-user through the on-boarding
```

- You can now try our beta Payments API
- Added new state `Queued` on transactions, for now this is a feature toggle on integrator level

> - One-time users can now claim there account during onboarding

Reviewer: ```suggestion
- One-time users can now claim their account during on-boarding
```

- Integrator can request all accounts, when a one-time user is onboarded

> - 

Reviewer: ```suggestion
```

### Fixed

> - Fixed that some ids on activity logs could be the same, result in some end-users are stuck when adding new accounts

Reviewer: ```suggestion
- Fixed that some ids on activity logs could be the same, resulting in some end-users getting stuck when adding new accounts
```

> - Unvalid paging token resulted in a `Internal Server Error`, now integreator gets a `Bad Request`

Reviewer: ```suggestion
-Invalid paging token resulted in a `Internal Server Error`, now integrator gets a `Bad Request`
```

> -Invalid paging token resulted in a `Internal Server Error`, now integrator gets a `Bad Request`

Reviewer: ```suggestion
- Invalid paging token resulted in a `Internal Server Error`, now integrator gets a `Bad Request`
```

> - If we could't send an end-user a email, an error message i shown

Reviewer: ```suggestion
- If we can't send an email to end-user, an error message is shown
```

> - Fixed browser navigation in persistent-user onboarding flow

Reviewer: ```suggestion
- Fixed browser navigation in persistent-user on-boarding flow
```
